src/SFML/Audio/SoundFileReaderOgg.cpp

```diff
@@ -36,7 +36,7 @@
 
 namespace
 {
-size_t read(void* ptr, size_t size, size_t nmemb, void* data)
+std::size_t read(void* ptr, std::size_t size, std::size_t nmemb, void* data)
```

Comment:
This function is used to fill an `ov_callbacks` structure whose read member is of type `size_t (*)(void*, size_t, size_t, void*)`.
See: https://www.xiph.org/vorbis/doc/vorbisfile/ov_callbacks.html
Wouldn't it be cleaner to keep using the exact same type?
When using C interfaces, I am not sure it makes sense to use `std::size_t`. What do you think?

Reply:
I don't really know what I prefer but I guess I'll remove the `std::` prefixes here.